tests/dragonfly/replication_test.py

```diff
@@ -1541,3 +1542,37 @@ async def test_df_crash_on_memcached_error(df_local_factory):
         memcached_client.set(b"key", b"data", noreply=False)
 
     await c_master.close()
+
```

Comment:
It would be beneficial to try to replicate different Dragonfly versions to check backward compatibility..

Reply:
It would :/ I checked it manually, no idea how to do it automatically with our current CI :confused: